src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -394,6 +431,9 @@ string Replacement(Match match)
 				command.Parameters.Add(dbParameter);
 			}
 
+			if (TimeoutLength != null)
+				command.CommandTimeout = (int)TimeoutLength;
```

Comment:
This should be moved to `CreateNewCommand` so that cached commands don't have to set it every time.